src/task/CleanStoppedSessionTask.ts

```diff
@@ -55,11 +55,6 @@ export default class CleanStoppedSessionTask extends BaseTask {
       this.context.turnCredentials = null;
       this.context.videoSubscriptions = null;
       this.context.transceiverController.reset();
-      this.context.mediaStreamBroker.releaseMediaStream(this.context.activeAudioInput);
```

Comment:
I feel like it should be part of the CleanStoppedSessionTask to clean up the media stream input and that should be the default behavior.
Could we instead pass in a flag to control whether we clean up the media stream input then pass reconnecting to it?

Reply:
Thanks for the suggestion.

- All child tasks accept only the `AudioVideoControllerState` context as a single parameter. Thus, I didn't add an extra flag in the constructor but abstract them into an explicit task.
- We can add a flag into the context, but it will then require to maintain the flag state. We wanted to avoid child tasks to know the session state.

Reply:
It makes sense to me. The only issue I have with this is now we need to know/remember cleanStoppedSesssionTask does not release media stream input and has to remember to call it seperately.

Reply:
The name `CleanStoppedSessionTask` is a general cleaning-up task, while `LeaveAndReceiveLeaveAckTask` and `ReleaseMediaInputTask` are specific tasks.

`LeaveAndReceiveLeaveAckTask` was also a part of `CleanStoppedSessionTask` but separated for having its own timeout.